backend/Recodo.BLL/Services/FolderService.cs

```diff
@@ -23,5 +26,50 @@ public async Task<FolderDTO> CreateFolder(NewFolderDTO newFolderDTO)
 
             return _mapper.Map<FolderDTO>(folderEntity);
         }
+
+        public async Task<List<FolderDTO>> GetFoldersByAuthorId(int authorId)
+        {
+            var foldersEntities = await _context.Folders.
+                Where(f => f.AuthorId == authorId).
+                ToListAsync();
+
+            if(foldersEntities.Count == 0)
+            {
+                throw new NotFoundException(nameof(Folder));
+            }
+
+            var folders = _mapper.Map<List<FolderDTO>>(foldersEntities);
+
+            return folders;
```

Comment:
Don't create variable if not needed
return _mapper..

Reply:
Agree, corrected